spec/fixtures/youtube_profiles.yml

```diff
@@ -5,3 +5,7 @@ whitehouse:
 nps:
   channel_id: natlparkservice_channel_id
   title: natlparkservice
+
+no_affiliate:
+  channel_id: noaffiliate_channel_id
+  title: noaffiliate_title
```

Comment:
Need to have a youtube_profile that does not have an affiliate to test the .active scopes out correctly.